lib/sql_footprint.rb

```diff
@@ -34,9 +34,11 @@ def exclude
       @capture = true
     end
 
-    def capture sql
-      return unless @capture && @filter.capture?(sql)
-      @statements.add @anonymizer.anonymize(sql)
+    private
+
+    def capturer database_name
```

Comment:
@brandonjoyce Can this be stored as a hash instead? 

The structure would look something like this:
```ruby
{
  db1: SqlCapturer.new(:db1),
  db2: SqlCapturer.new(:db2),
}
```

We can even take advantage of giving Hash a default value block like this:
```ruby
@capturers = Hash.new { |hash, database_name| hash[database_name] = SqlCapturer.new(database_name) }
```